### ISSUE TITLE
[Docs] Extensibility: New page needed for CLI Plugins

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery#12181]
### Current State
There are pages/sections in Meshery Docs dedicated to describing the various types of Meshery Extensions, but not a page dedicated for CLI plugins.

### Desired State
1. A new page with general explanation.
2. Listing of github.com/meshery/helm-kanvas-snapshot and explanation of plugin.
3. Listing of github.com/meshery/kubectl-kanvas-snapshot and explanation of plugin.
4. Listing of github.com/meshery/kubectl-meshsync-snapshot and explanation of plugin.

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by leecalcote** [comment]
@Alero-Awani @Aijeyomah

**Comment by divyansh2375** [comment]
hi @leecalcote  can i work on this issue under hacktoberfest?

**Comment by leecalcote** [comment]
@divyansh2375, yes, you'll be successful in at least getting the first two documented. It's ok to break this issue into three PRs (only two of which you might attempt given the information readily available in written form). If you were to watch one of Meshery Development meeting livestreams, there is a verbal explanation as to what the third is.

**Comment by Tharanishwaran** [comment]
@leecalcote can you assign me this issue ?

**Comment by matrixkavi** [comment]
Closing this issue as done as the following pages exist in current docs page:

https://docs.meshery.io/extensions/kubectl-meshsync-snapshot/
https://docs.meshery.io/extensions/helm-kanvas-snapshot/
https://docs.meshery.io/extensions/kanvas-snapshot/
https://docs.meshery.io/extensions/
